crates/turborepo-lib/src/daemon/mod.rs

```diff
@@ -11,4 +11,12 @@ pub use server::{serve, CloseReason};
 
 pub(crate) mod proto {
     tonic::include_proto!("turbodprotocol");
+
+    /// The version of the protocol that this library implements.
+    ///
+    /// Protocol buffers aim to be backward and forward compatible at a protocol
+    /// level, however that doesn't mean that our daemon will have the same
+    /// logical API. We may decide to change the API in the future, and this
+    /// version number will be used to indicate that.
+    pub const VERSION: &str = "1.10.17";
```

Comment:
Just because this is confusing and it took me awhile to convince myself this is correct can we maybe focus this comment (or add to it) exactly when bumping this should happen and what major/minor/patch bumps indicate. I don't think this needs to be 3rd-party facing docs yet, just guidance for ourselves.